mantidproject/mantid
# Add Documentation for new Model Fitting and Corrections tabs

**Description of work.**
This PR adds documentation pages for the new Model fitting tab, and Corrections tab in the Muon interfaces. It also updates the Fitting and Sequential Fitting documentation pages, and adds better release notes for these new features.

**To test:**
Spell and grammar check
Build docs-html
Make sure the new links work
Make sure the images are displayed as expected 

*There is no associated issue.*

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
